operator/internal/manifests/internal/rules/marshal_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	lokiv1 "github.com/grafana/loki/operator/apis/loki/v1"
 	"github.com/grafana/loki/operator/internal/manifests/internal/rules"
 	"github.com/stretchr/testify/require"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Following k8s pkg import conventions.